swe_list.m

```diff
@@ -90,11 +90,9 @@
 % swe_list characterizes SwE data (thresholded at u and k) in terms of
 % excursion sets (a collection of face, edge and vertex connected subsets
 % or clusters).  The corrected significance of the results are based on
-% set, cluster and voxel-level inferences using distributional
-% approximations from the Theory of Gaussian Fields and/or common bootstrap 
-% methods. These distributions assume that the SwE map is a reasonable 
-% lattice approximation of a continuous random field with known component 
-% field smoothness.
+% voxel-level inferences assuming Gaussian errors, uncorrected or 
```

Comment:
@BryanGuillaume actually, this was the only change I made here... please review and see if you agree.